packages/blocks/src/root-block/commands/model-crud/get-selected-models.ts

```diff
@@ -7,9 +7,11 @@ export const getSelectedModelsCommand: Command<
   'selectedModels',
   {
     types?: Extract<BlockSuite.SelectionType, 'block' | 'text' | 'image'>[];
+    mode?: 'all' | 'flat' | 'highest';
```

Comment:
Since this should be part of our public commands API, could you clarify the mode definition in the `/** ... */` comment?